plateWars.lua

```diff
@@ -365,7 +406,11 @@ function plateWarsDefusedTimer(pid, cellDescription, uniqueIndex)
 end
 
 function plateWars.hasBomb(pid)
-    return plateWars.bomb.baseData.carrierPid == pid
+    -- return plateWars.bomb.baseData.carrierPid == pid
+    if inventoryHelper.containsItem(Players[pid].data.inventory, plateWars.bomb.refIds.inventoryItem) then
```

Comment:
put this back here for bomb testing purposes before i noticed the addbomb command

Reply:
I'd like to stick to the previous version of the function that doesn't iterate player's inventory. We could - for testing purposes - make the carrierPid a table and have the removeBomb function delete pid from that table instead.